schematools/permissions.py

```diff
@@ -41,13 +44,24 @@ def revoke_permissions(engine, role):
 
 
 def apply_schema_and_profile_permissions(
```

Comment:
Het zal beter zijn om deze functie in 2 te splitsen, om scope beperkt te houden.

Reply:
Die dubbele taak die hier uitgevoerd wordt, dat laat ik even zo omdat ze in dezelfde transaction moeten. Als we later verder gaan met de Profiles, dan pak ik dat aan.